array.h

```diff
@@ -64,10 +64,16 @@ ostream &operator<<(ostream &os, CArray<T> &obj){
     return os;
 }
 
-// TODO
 template <typename T>
 istream & operator>>(istream &is, CArray<T> &obj){
-    // TODO
+    size_t vcount;
+    is >> vcount;
+    
+    T value;
+
+    while(is>>value)
+        obj.insert(value);
```

Comment:
I guess it's OK for the first homework, but you should keep in mind call a read function inside the class, and inside it
reset the object in order to clean past values.